### DISCUSSION TITLE
How to download the 'obda-to-r2rml' script and is there any related documentation?

### DISCUSSION THREAD

**Discussion by chenkuncloud** [ontop/ontop#611] [Q&A] [answered]
Hi,

I know that Ontop provides a script that can implement `obda-to-r2rml`, but I haven't found the download address and related documents.

Could you please provide it to me?

**Comment by bcogrel** [comment, marked as answer]
It is part of the Command Line Interface (see [`ontop mapping to-r2rml`](https://ontop-vkg.org/guide/cli.html#ontop-mapping-to-r2rml)).

You can download it here:  https://github.com/ontop/ontop/releases/tag/ontop-5.0.1

**Reply by chenkuncloud** [reply]
Thanks
